libsolidity/ast/Types.h

```diff
@@ -319,6 +319,8 @@ class Type: private boost::noncopyable, public std::enable_shared_from_this<Type
 class AddressType: public Type
 {
 public:
+    static AddressType& address() { static std::shared_ptr<AddressType> addr(new AddressType()); return *addr; }
```

Comment:
Uses spaces for indentation.

Reply:
Also please don't use `new`. Use `make_shared` instead.

Reply:
Am I not using spaces for indentation? My editor is set to use 4 spaces for tabs, I'm trying to change that line but I think it's already spaces

Reply:
I'm sorry, this was a little unclear: You should use tabs everywhere, but these are spaces.

Reply:
Got it